src/main.cpp

```diff
@@ -143,6 +143,8 @@ static void lPrintVersion() {
     printf("        force-aligned-memory\t\tAlways issue \"aligned\" vector load and store instructions\n");
     printf("        reset-ftz-daz\t\t\tReset FTZ/DAZ flags on ISPC extern function entrance / restore on return\n");
     printf("    [--pic]\t\t\t\tGenerate position-independent code.  Ignored for Windows target\n");
+    printf("    [--PIC]\t\t\t\tGenerate position-independent code avoiding any limit on the size of the flobal offset "
```

Comment:
```suggestion
    printf("    [--PIC]\t\t\t\tGenerate position-independent code avoiding any limit on the size of the global offset "
```

Reply:
I use this plugin and it helps a lot against this kind of typos: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
done